test.ssl/Makefile

```diff
@@ -0,0 +1,27 @@
+.PHONY: all clean
```

Comment:
Can these files be placed in e.g. test/ssl/ so that all test related files are under test/ ? Less cluttered directory structure IMO.